src/python/grpcio/grpc/aio/_base_call.py

```diff
@@ -44,6 +44,7 @@ def cancelled(self) -> bool:
 
         Returns:
           A bool indicates whether the RPC is cancelled or not.
+
```

Comment:
I wonder what resulted in these lines added everywhere. Is this the result of black-compatible formatting?